src/blackbird.g4

```diff
@@ -58,7 +58,7 @@ measure             : MEASURE;
 
 //  function arguments
 
-arguments           : (LBRAC (val_list+=val (COMMA val_list+=val)*)? (kwarg_list+=kwarg (COMMA kwarg_list+=kwarg)*)? RBRAC);
```

Comment:
This is they key place where the bug was fixed?

Reply:
yep! I forgot to specify a comma inbetween positional and keyword args